UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -89,9 +89,34 @@ public async Task WhenRequestUploadFileShareServiceData_ThenReturnsCloudBlockBlo
 
             A.CallTo(() => fakeazureBlobStorageClient.GetCloudBlockBlob(A<string>.Ignored, A<string>.Ignored, A<string>.Ignored)).Returns(new CloudBlockBlob(new System.Uri("http://tempuri.org/blob")));
 
-            var result = await fulfilmentFileShareService.UploadFileShareServiceData(uploadFileName,new List<FulfillmentDataResponse>(), connectionString, containerName);
+            var result = await fulfilmentFileShareService.UploadFileShareServiceData(uploadFileName,new List<FulfilmentDataResponse>(), connectionString, containerName);
 
             Assert.AreEqual("http://tempuri.org/blob", result);
         }
+
+        [Test]
+        public void WhenRequestDownloadFileShareServiceFiles_ThenReturnsFileToSpecificPath()
```

Comment:
Can we rename it from WhenRequestDownloadFileShareServiceFiles_ThenReturnsFileToSpecificPath() to WhenValidRequest_ThenDownloadFileShareServiceFilesReturnsFile()

Similarly change WhenRequestDownloadFileShareServiceFiles_ThenReturnsNoFileToSpecificPath() to
WhenInvalidRequest_ThenDownloadFileShareServiceFilesReturnsNoFile()

Reply:
Renamed Unit Test method name